harvard-lts/drs-migration-dashboard
# Removing /piechart/ from url.

**Removing /piechart/ from url.**
* * *

**JIRA Ticket**: [14](https://github.com/harvard-lts/drs-migration-dashboard/issues/14)


# What does this Pull Request do?
This changes the URL of the migration status page from `/migrationstatus/piechart/` to just `/migrationstatus/` 

# How should this be tested?

A description of what steps someone could take to:
* Go to https://localhost:3001/migrationstatus/
* The charts and graphs should all appear as usual.
* Go to https://localhost:3001/migrationstatus/piechart/
* This should redirect you to https://localhost:3001/migrationstatus/ with a 301 (moved permanently) status.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No 
- integration tests? No

# Interested parties
@awoods @mferrarini @dl-maura @enriquediaz
